### DISCUSSION TITLE
Centos7 install/run issue (NOT via docker)

### DISCUSSION THREAD

**Discussion by exit151** [Lissy93/dashy#739] [Help and Support] [answered]
So I followed the guide (mostly, I launch it with the screen app to keep the terminal usable) and it's not launching.
my script file is as follows:  screen -d -m -S dashy yarn start
working fine, no errors (shows checking config, green msg saying config file is valid, no issues found)
Tells me the dashy is up and gives me the loopback ip (I don't use this since it's on a dedicated vm server).

So on another PC I go to the site at the 4000 port it picked, it says a few messages with a fancy Dashy loading screen..  The ones I recall, in order are:
building
finishing off
almost done
not loong left
taking longer than expected
possible error, check logs

...and that's it.  No mention of where log files might be.  The website is great - if you're using Docker - but doesn't me the faintest hint at where I might find the logs.  I do have the SSL Not enabled message (under the config file is valid), but read elsewhere that's not a concern, just an informational piece that I have not set up SSL for this).  I guess question one would be to confirm this is true, and SSL is not required to run dashy?  And question two will be where might I find said logs to see what's going on, since the console  isn't reporting any errors on it (last thing it states is Dashy is Up-to-Date)

Thanks in advance, 

exit151

**Comment by lissy93** [comment, marked as answer]
I only just saw this sorry...
If you're running on bare metal, just run `yarn build`. That should fix you're issue.
